substratevm/src/com.oracle.svm.agent/src/com/oracle/svm/agent/NativeImageAgentJNIHandleSet.java

```diff
@@ -26,10 +26,13 @@
 
 import static com.oracle.svm.jni.JNIObjectHandles.nullHandle;
 
+import com.oracle.svm.jni.nativeapi.JNIFieldId;
 import com.oracle.svm.jvmtiagentbase.JNIHandleSet;
 import com.oracle.svm.jni.nativeapi.JNIEnvironment;
 import com.oracle.svm.jni.nativeapi.JNIMethodId;
 import com.oracle.svm.jni.nativeapi.JNIObjectHandle;
+import com.oracle.svm.jvmtiagentbase.Support;
+import org.graalvm.compiler.lir.util.GenericValueMap;
 
 public class NativeImageAgentJNIHandleSet extends JNIHandleSet {
```

Comment:
Eventually, we should add frequently used `JNIMethodId` and `JNIObjectHandle` values in the new code to this class, ideally initializing them lazily, so we can avoid repeated lookups by name.

Reply:
I have moved them into `NativeImageAgentJNIHandleSet.java`.